lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb

```diff
@@ -57,6 +57,29 @@ module ActiveModel
       #
       # #### Qualifiers
       #
+      # ##### allow_nil
+      #
+      # Use `allow_nil` if you want to validate a `belongs_to` association but
```

Comment:
Does this have to be a `belongs_to`, can't it be any attribute?

I would use the same description we use for the other instances of `allow_nil`, which is:

> Use `allow_nil` to assert that the attribute allows nil.